src/AElf.Contracts.TestKit.AEDPoSExtension/BlockMiningService.cs

```diff
@@ -47,6 +53,7 @@ public BlockMiningService(IServiceProvider serviceProvider)
             _contractTesterFactory = serviceProvider.GetRequiredService<IContractTesterFactory>();
```

Comment:
we should improve inject the exact service instead of IServiceProvider, improve this on other PR